### ISSUE TITLE
quarto install tinytex leads to deno panicking

### ISSUE THREAD

**Issue by Obscrts** [denoland/deno#25002]
Hey there,
by inserting the command "quarto install tinytex" I get the following error

============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: windows x86_64
Version: 1.41.0
Args: ["C:\\Program Files\\Quarto\\bin\\tools\\x86_64\\deno", "run", "--unstable-ffi", "--no-config", "--cached-only", "--allow-read", "--allow-write", "--allow-run", "--allow-env", "--allow-net", "--allow-ffi", "--v8-flags=--enable-experimental-regexp-engine,--max-old-space-size=8192,--max-heap-size=8192", "--importmap", "C:\\Program Files\\Quarto\\bin\\vendor\\import_map.json", "C:\\Program Files\\Quarto\\bin\\quarto.js", "install", "tinytex"]

thread 'main' panicked at cli\args\mod.rs:596:14:
Failed to add platform cert to root cert store: InvalidCertificate(BadEncoding)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by lucacasonato** [comment]
Duplicate of #24137

**Comment by RRMaximiliano** [comment]
I was able to install TinyTeX using Quarto by following these steps:

1. **Upgrade Deno**: Run `deno --upgrade` to upgrade your Deno installation. Note that Quarto may not use this upgraded version immediately. In my case, I upgraded to version 1.46.1, whereas the original Deno version installed by Quarto was 1.41.

2. Move the `deno.exe` file from the upgraded installation to the Quarto directory. For my setup, this directory was: 

   Upgraded deno location
   ```
   C:\Users\ifyou\.deno
   ```
   Move to:
   ```
   C:\Users\ifyou\AppData\Local\Programs\Quarto\bin\tools\x86_64
   ```

3. After moving `deno.exe`, I was able to successfully install TinyTeX and other extensions.
